recipes/jupyter-book/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "jupyter-book" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 77da2c5714494d0a1eed494542f1a717a7addbb0dfa30b1f7292f7a4caf0fb0c
+
+build:
+  number: 0
+  entry_points:
+    - jb = jupyter_book.commands:main
+    - jupyter-book = jupyter_book.commands:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - click
+    - docutils >=0.15
+    - jsonschema
+    - myst-nb >=0.10.1,<0.11
+    - nbconvert <6
+    - nbformat
+    - python >=3.6
+    - pyyaml
+    - setuptools
+    - sphinx >=2,<4
+    - sphinx-comments
+    - sphinx-copybutton
+    - sphinx-panels >=0.5.0,<0.6
+    - sphinx-thebe >=0.0.6
+    - sphinx-book-theme >=0.0.36
+    - sphinx-togglebutton
+    - sphinxcontrib-bibtex <2
+
+test:
+  imports:
+    - jupyter_book
+    - jupyter_book.commands
+    - jupyter_book.directive
+  commands:
+    - jb --help
+    - jupyter-book --help
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: "https://executablebooks.org/"
+  license: BSD-3-Clause
+  license_file: LICENSE
+  summary: "Jupyter Book: Create an online book with Jupyter Notebooks"
+  dev_url: https://github.com/executablebooks/jupyter-book
+  doc_url: https://jupyterbook.org
+
+extra:
+  recipe-maintainers:
+    - chrisjsewell
+    - moorepants
```

Comment:
Feel free to add me as well!

Reply:
Done

Reply:
I'm sure it goes without saying, but I would just ask you guys to be vigilant with dependency updates when merging new versions 😉

Reply:
Given the complexity of the dependency pins of this package, it would be smart to add some tests that ensure some basic functionality works from the built package. Then tests would fail on updates. But maybe `pip check` is sufficient?

Reply:
Also, with other feedstocks I maintain, I rely wholly on upstream's dependency specifications. So if jupyter-book does a tight job managing this in their setup.py it should work out in the translation.

Reply:
> So if jupyter-book does a tight job managing this in their setup.py it should work out in the translation.

yes indeed but conda-forge does not automate dependency updates in their automated version PRs

Reply:
Yes, we feedstock maintainers are well aware of that. It is so much easier to maintain the feedstock dependency pins when upstream does a nice job of it. I deal with some others where that isn't the case, so it takes quite a bit of time on each feedstock update to manually check everything.

Reply:
https://github.com/executablebooks/jupyter-book/issues/1141